fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReferenceTest.java

```diff
@@ -69,18 +81,25 @@ public String createPatientWithReference(String reference) throws Exception {
     public void testCreatePatient() throws Exception {
         // references without an explicit type will be interpreted as literals (e.g. externals) not local references
         // we will not infer their type, which means that searches specifying a type will not match them by design
-        patientResourceIdWithReferenceId = createPatientWithReference("3001");
+        patientResourceIdWithReferenceId = createPatientWithReference("organization", "3001");
```

Comment:
For consistency, we may want to mark this in the proper case for a FHIR Resource 'Organization'

Reply:
This parameter actually refers to a search parameter code, so I think case is appropriate.